### PR TITLE
Adding L0 tests for GitHub release task

### PR DESCRIPTION
**Pull request by mdmdakbar**

### CODE REVIEW

File: Tasks/GitHubReleaseV0/Tests/UtilityL0Tests.ts [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        parsedHttpHeaderLink = Utility.parseHTTPHeaderLink(headerLink);
+
+        if (JSON.stringify(parsedHttpHeaderLink) === JSON.stringify(expectedParsedHeaderLink)) {
+            console.log(TestString.parseHTTPHeaderLinkKeyword);
+        }
+    }
+
+    public static validateExtractRepositoryOwnerAndName() {
+        let repoName = "owner_name/repo_name";
+
```
Comment (Ajay-MS) [inline Tasks/GitHubReleaseV0/Tests/UtilityL0Tests.ts]: remove necessary lines. Applicable at multiple places.

File: Tasks/GitHubReleaseV0/Tests/TestStrings.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
+
+export class TestString {
+    public static readonly createReleaseSuccessKeyword = "CreateReleaseSuccess";
```
Comment (Ajay-MS) [inline Tasks/GitHubReleaseV0/Tests/TestStrings.ts]: Inconsistency - Few variables are defined with type while few are not.

File: Tasks/GitHubReleaseV0/operations/Utility.ts [hunk 3]
```diff
@@ -9,13 +9,22 @@ export class Utility {
...
+        if (!!githubEndpointObject) {
+            tl.debug("Endpoint scheme: " + githubEndpointObject.scheme);
+            
+            if (githubEndpointObject.scheme === 'PersonalAccessToken') {
+                githubEndpointToken = githubEndpointObject.parameters.accessToken
+            } else if (githubEndpointObject.scheme === 'OAuth'){
+                // scheme: 'OAuth'
+                githubEndpointToken = githubEndpointObject.parameters.AccessToken
+            }
+            else if (githubEndpointObject.scheme) {
```
[the remark this answers is not in the document]
Comment (mdmdakbar) [reply Tasks/GitHubReleaseV0/operations/Utility.ts]: Scheme can't be null if endpoint is valid. IF endpoint is invalid then githubEndpointObject will be null or undefined in that case

File: Tasks/GitHubReleaseV0/operations/Constants.ts [hunk 4]
```diff
@@ -0,0 +1,19 @@
+
+export class Inputs {
```
[the remark this answers is not in the document]
Comment (mdmdakbar) [reply Tasks/GitHubReleaseV0/operations/Constants.ts]: Will take in next PR

File: Tasks/GitHubReleaseV0/Tests/UtilityL0Tests.ts (hunk 1 above)
Comment (mdmdakbar) [reply Tasks/GitHubReleaseV0/Tests/UtilityL0Tests.ts]: Will take in next PR

File: Tasks/GitHubReleaseV0/Tests/TestStrings.ts (hunk 2 above)
Comment (mdmdakbar) [reply Tasks/GitHubReleaseV0/Tests/TestStrings.ts]: Will take in next PR

File: Tasks/GitHubReleaseV0/operations/Constants.ts (hunk 4 above)
[the remark this answers is not in the document]
Comment (mdmdakbar) [reply Tasks/GitHubReleaseV0/operations/Constants.ts]: we may add some more constant classes here in future.

**Pull request merged**